xml/soc-operations-maintenance.xml

```diff
@@ -482,6 +482,109 @@
     </listitem>
    </itemizedlist>
   </section>
+
+  <section xml:id="sec-upgrade-unsupported-configs">
+   <title>Unsupported configurations</title>
+   <para>
+    In &cloud; 9, certain configurations and barclamp combinations are no
+    longer supported. See the <link
+    xlink:href="https://www.suse.com/releasenotes/x86_64/SUSE-OPENSTACK-CLOUD/9/">&cloud;
+    9 release notes</link> for details. The upgrade prechecks fail if they
```

Comment:
The pdf shows the URL as visible part of the text, is that correct?

Reply:
Yes, PDF show the full link (useful when you print docs).